src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,48 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_CAPACITY = 10;
+    private final Pair<K, V>[] pairs = new Pair[MAX_STORAGE_CAPACITY];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
int value is 0 by default